src/components/Inputs/ColoredLabelInput.tsx

```diff
@@ -81,7 +83,7 @@ const InputStyled = styled(Input)`
     theme.name === 'dark' ? color : tinycolor(color).isLight() ? theme.font.primary : theme.font.contrastPrimary};
 
   &:not([value='']) {
-    padding-left: 20px;
+    padding-left: calc(1rem + 0.2rem + 1em);
```

Comment:
Is this a magic number? 😅  I guess this is the summary of paddings and spacings of other elements, right? Which means if they get changed elsewhere they'd need to be changed here as well... Maybe exported vars would help here?

Reply:
It's other element widths yeah - I figured it's at the very least better than the old value. I think it's unnecessary "busy work" to make this more dynamic but sure if you think otherwise :+1:

Reply:
Well, it's true that we don't rly have a fixed system of spacings with shared variables. But I still think it's worth to create vars when component spacings are so interdependent.